gating/pre_merge_test/run_system_tests.sh

```diff
@@ -21,28 +21,38 @@ export SYS_INVENTORY="/opt/openstack-ansible/playbooks/inventory"
 
 ## Main ----------------------------------------------------------------------
 
-
-# 1. Clone test repo into working directory.
+# 1. Clone test repository into working directory.
```

Comment:
`-e` needs to be set here for this section to fail hard if an error is encountered prior to step # 2.